app-sso/how-to-guides/service-operators/cors.hbs.md

```diff
@@ -66,6 +66,46 @@ Requirements for allowed origin designations include:
 - When using `allowAllOrigins`, you must explicitly set the annotation `sso.apps.tanzu.vmware.com/allow-unsafe-cors: ""`.
   This is an acknowledgement that using `allowAllOrigins` is inherently unsafe.
 
+## <a id="additional-cors-configuration"></a> Additional CORS configuration
```

Comment:
**[suggestion]:** The premise is to unlock the `client_credentials` grant for single-page apps. Help the service operator to accomplish that specifically in addition to the general documentation of the new fields.

Reply:
Good idea. Added a blurb re:client_credentials, including "it's not a recommended practice".

Note: the new defaults mean that authserver should now support "default" `client_credentials` grants out of the box (default = `Access-Control-Request-Method: POST` + `Access-Control-Request-Headers: Authorization`).

The additional configuration options don't specifically help the `client_credentials` flow, hence the initial lack of docs. We are mostly adding extra knobs for uses-cases we have not foreseen.